search.js

```diff
@@ -72,6 +72,7 @@ var button = document.getElementById("submit");
 button.addEventListener("click", function(ev) {
   ev.preventDefault();
   var damageValue = document.getElementById("damage").value;
+  if (!!!damageValue) return false;
```

Comment:
Using `!!!damageValue` also returns false with value 0.

```suggestion
  if (damageValue === '') return false;
```

Reply:
oh I'm sorry, i assumed all pokemon cards have an attack value :)